Release/include/cpprest/containerstream.h

```diff
@@ -399,7 +399,7 @@ class basic_container_buffer : public streams::details::streambuf_state_manager<
         auto readBegin = std::begin(m_data) + m_current_position;
         auto readEnd = std::begin(m_data) + newPos;
 
-#ifdef _WIN32
+#if defined(_WIN32) && (_ITERATOR_DEBUG_LEVEL != 0)
```

Comment:
It seems like maybe `#if defined(_ITERATOR_DEBUG_LEVEL) && _ITERATOR_DEBUG_LEVEL != 0` might be a more targeted way to say this?